devito/operator/profiling.py

```diff
@@ -52,7 +55,13 @@ def __init__(self, name):
         # Python-level timers
         self.py_timers = OrderedDict()
 
-        self.initialized = True
+        self._attempted_init = True
+
+    def add_include_dir(self, dir_path):
```

Comment:
you don't need these public methods since the updated attributes (`_include_dirs` , `_lib_dirs`) are only updated internally from the class itself, not from the outside